scripts/load_development_data

```diff
@@ -55,6 +55,10 @@ function load_dev_data() {
     docker-compose \
         exec -T django ./manage.py setup config/config.xml -t
 
+    echo "Creating database views"
+    docker-compose \
+        exec -T django ./manage.py setup config/config.xml --views
```

Comment:
Sorry for the drive-by but I also wound up doing this if you can believe it 🤦‍

Reply:
I'll rebase out before merging. 👍